src/SlmLocale/Service/PrimaryLanguageHelperFactory.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Copyright (c) 2012-2019 Jurian Sluiman.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ *   * Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ *
+ *   * Redistributions in binary form must reproduce the above copyright
+ *     notice, this list of conditions and the following disclaimer in
+ *     the documentation and/or other materials provided with the
+ *     distribution.
+ *
+ *   * Neither the names of the copyright holders nor the names of the
+ *     contributors may be used to endorse or promote products derived
+ *     from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ * @author      Jurian Sluiman <jurian@juriansluiman.nl>
+ * @copyright   2012-2013 Jurian Sluiman.
+ * @license     http://www.opensource.org/licenses/bsd-license.php  BSD License
+ * @link        http://juriansluiman.nl
+ */
+namespace SlmLocale\Service;
+
+use Interop\Container\ContainerInterface;
```

Comment:
I think this can be `Psr\Container\ContainerInterface` as we require ServiceManager ^3.2

Reply:
`Zend\ServiceManager\Factory\FactoryInterface` is still using `Interop\Container\ContainerInterface`

Reply:
hmm, it looks we need to update servicemanager because they support PSR only from [3.4.0](https://github.com/zendframework/zend-servicemanager/blob/master/CHANGELOG.md#340---2018-12-22) but I guess if this is final we can change this anytime.